projects/ccf-rui/src/app/shared/components/organ-selector/organ-selector.component.spec.ts

```diff
@@ -92,4 +92,13 @@ describe('OrganSelectorComponent', () => {
     expect(instance.timeoutHandler).toBeUndefined();
   });
 
+  it('should shift the carousel when scroll is called', async () => {
+    const { instance } = await shallow.render();
+    instance.onLeft = false;
+    const spy = spyOn(instance, 'shift');
+    setTimeout(() => {
```

Comment:
This will not work. If you want to wait x milliseconds do:

Helper function
```
function wait(duration: number): Promise<void> {
  return new Promise(resolve => setTimeout(resolve, duration));
}
```

Then you can simply `await wait(2000);`